src/portfft/descriptor.hpp

```diff
@@ -592,7 +653,7 @@ class committed_descriptor {
       std::vector<IdxGlobal> sub_batches;
       std::vector<IdxGlobal> inclusive_scan;
       std::size_t cache_required_for_twiddles = 0;
-      for (const auto& kernel_data : dimensions.at(global_dimension).kernels) {
+      for (const auto& kernel_data : dimensions.at(global_dimension).forward_kernels) {
```

Comment:
Why do we not need to look at backwards kernels too?

Reply:
This part only deals with the factor sizes, and not the direction. 
Since the factorization is the exact same, we can refer to only one of them

Reply:
Nice! If either direction failed to compile, we'd already have failed, right?

Reply:
Correct, it's ensured that both are built